src/init.rs

```diff
@@ -158,6 +158,12 @@ if [[ ${preexec_functions[(ie)starship_preexec]} -gt ${#preexec_functions} ]]; t
     preexec_functions+=(starship_preexec);
 fi;
 STARSHIP_START_TIME="$(date +%s)";
+function zle-keymap-select
+{
+    PROMPT=$(starship prompt --keymap=$KEYMAP);
+    zle reset-prompt;
+}
```

Comment:
Whoops, need a semicolon after the function close brace.
```suggestion
};
```

(The rules are dumb, this is why we're trying to get rid of one-line eval)

Reply:
oh. sorry.
I'll squash this to the previous commit.